From 562dbef8dd934efc06dc99d21e8f8bfa732cae96 Mon Sep 17 00:00:00 2001
From: Jim Bride <jim.bride@intel.com>
Date: Fri, 10 Feb 2012 09:41:56 -0800
Subject: Add lsusb to the list of supported utilities.

Add lsusb to the list of supported utilities so a symlink to
toolbox named lsusb is generated in /system/bin.

Change-Id: I16d23c82d23867b17f23c24682bf2117e9400907
Signed-off-by: Jim Bride <jim.bride@intel.com>
---
 toolbox/Android.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/toolbox/Android.mk b/toolbox/Android.mk
index c764690..ab2fc85 100644
--- a/toolbox/Android.mk
+++ b/toolbox/Android.mk
@@ -57,6 +57,7 @@ TOOLS := \
 	touch \
 	lsof \
 	du \
+	lsusb \
 	md5 \
 	clear \
 	getenforce \
-- 
1.7.4.1

